src/test/java/com/micetr0/model/MeasureTest.java

```diff
@@ -0,0 +1,11 @@
+package com.micetr0.model;
+
+import org.junit.Test;
+import com.micetr0.model.Note;
+
+import static org.junit.Assert.*;
+
+public class MeasureTest {
+//    private Note note = new Note();
```

Comment:
need test cases

Reply:
Measure still not implemented. Will be addressed in future pull request.